src/vs/workbench/browser/parts/editor/editorGroupsControl.ts

```diff
@@ -37,6 +37,8 @@ import { attachProgressBarStyler } from 'vs/platform/theme/common/styler';
 import { IDisposable, combinedDisposable } from 'vs/base/common/lifecycle';
```

Comment:
This is the only file I did not yet review. Will first try out the feature a bit.